bylaws.tex

```diff
@@ -251,8 +251,9 @@
 The Membership Evaluation process occurs once per academic year.
 It is performed as part of the Evaluation Process that takes place during the spring semester to comply with RIT Housing deadlines.
 \bsubsubsection{Voting}
-A quorum of members, on a member by member basis, ensures that each Active Member has met the minimum requirements in order to continue as an Active Member in the following year.
-All Active Members will be held to the requirements and evaluated unless they have received an exemption from the Executive Board prior to the event.
+A quorum of members, on a member by member basis, ensures that each Active Member has met the minimum objective requirements in order to continue as an Active Member in the following year. Any Active Member who has met all the minimum objective requirements will be added to a batch that will not be voted on and no member can be pulled. This batch will not be eligable to be voted on and all members in this batch will pass the membership evaluations in the beginning of the meeting.
```

Comment:
Keep to the standard. Newline after each sentence

Reply:
ok